src/main/kotlin/app/revanced/patches/youtube/layout/hideendscreencards/resource/patch/HideEndscreenCardsResourcePatch.kt

```diff
@@ -30,8 +30,8 @@ class HideEndscreenCardsResourcePatch : ResourcePatch {
                 "revanced_hide_endscreen_cards",
                 StringResource("revanced_hide_endscreen_cards_title", "Hide end-screen cards"),
                 true,
-                StringResource("revanced_hide_endscreen_cards_summary_on", "End-screen cards are hidden"),
-                StringResource("revanced_hide_endscreen_cards_summary_off", "End-screen cards are shown")
+                StringResource("revanced_hide_endscreen_cards_summary_on", "Video overlay end-screen cards are hidden"),
```

Comment:
Nowhere else "Video overlay" is used. For consistency and simplicity you shouldn't just introduce it here, same for the info cards.